README.rst

```diff
@@ -83,7 +92,7 @@ Features include
 **Actions:** This actions can be applied when a condition is met:
 
 * Display a message
-* Redirect to another unit in the same subsection (without reloading the page)
+* Redirect to another unit in the same subsection (without reloading the page) :warning:[DEPRECATED: Since version 2.0.0 and not working with MFE Learning]
```

Comment:
```suggestion
* Redirect to another unit in the same subsection (without reloading the page) :warning:[DEPRECATED: Since version 2.0.0 and not working with the Learning MFE]
```